### ISSUE TITLE
Quest Grand Theft Palomino Death Knight

### ISSUE THREAD

**Issue by Piksna007** [dalaranwow/dalaran-wow#4389]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: In the quest Grand Theft Palomino from the quests for the Death Knight, where you get a mount "horse", you go and steal a horse from the stable and when you get to the npc "Salanar the horseman" where you should finish the quest, when you get off the horse the npc that we should complete the quest kills the horse. And the worst problem is that we cant get any further quests because of this one, I finished all the quests and this is the only one left and I am stuck at lvl 56 with Death Knight without all the quests.

**Expected behaviour**: What it must happen is, when you get the horse to the npc where we finish the quest, it should be complete when we bring and get off the horse.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
tghen it's a phasing issue. pls let a gamemaster recheck you current progress in the questline + let him move you to the correct phase in the game.

Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
